flatdata-rs/rustfmt.toml

```diff
@@ -1,3 +1,3 @@
-wrap_comments = true
```

Comment:
Are you sure you want to remove it? This is actually very useful.

Reply:
It broke some long links in the docs... do you know if I can annotate the appropriate places to prevent it from doing that?

Reply:
#[rustfmt::skip] does not seem to work for the docs at the top of lib.rs

Reply:
Hmm, seems like a bug in rustfmt. Which place did break exactly?

Reply:
Added back and just excluded the problematic file